### PR TITLE
Download Edge from universal packages, extra stage to run brokerhost tests in broker CI

### PR DESCRIPTION
**Pull request by fadidurah**
Add a step to download edge apk from universal packages. This lets us use LocalApk installation during our UI automation. Also, create an extra stage to run brokerhost tests in Broker CI Pipeline https://dev.azure.com/IdentityDivision/Engineering/_build?definitionId=1490&_a=summary

### CODE REVIEW

File: azure-pipelines/ui-automation/broker-test.yml [hunk 1]
```diff
@@ -102,6 +105,17 @@ stages:
         msalFlavor: Dist
         brokerSource: LocalApk
         brokerUpdateSource: LocalApk
+# msalautomationapplocalbrokerhost
+- stage: 'msalautomationapplocalbrokerhost'
+  dependsOn: []    # this removes the implicit dependency on previous stage and causes this to run in parallel
+  displayName: Build MSAL Automation APKs (LocalBrokerHost)
```
Comment (somalaya) [inline azure-pipelines/ui-automation/broker-test.yml:111]: Fadi, what is the difference between msalautomationapp and msalautomationapplocalbrokerhost?  I did not understand what LocalBrokerHost means here.

**Review by somalaya — APPROVED** [review]

File: azure-pipelines/ui-automation/broker-test.yml (hunk 1 above)
Comment (fadidurah) [reply azure-pipelines/ui-automation/broker-test.yml:111]: I'm creating another msal automation app with Local BrokerHost Variant. This is needed for some test cases to pass without needing to create another pipeline.

Comment (somalaya) [reply azure-pipelines/ui-automation/broker-test.yml:111]: Ok.. so this just means that msal automation app would run with broker as brokerHost app right?
I think we should mention that in one of the fields as "MSAL Automation app with broker set as local brokerHost app" and we should also update the PR description to indicate this change

Comment (fadidurah) [reply azure-pipelines/ui-automation/broker-test.yml:111]: Yeah, these changes came in after I initially made the Pr. Let me make some changes

Comment (fadidurah) [reply azure-pipelines/ui-automation/broker-test.yml:111]: The display name does include a detail mentioning BrokerHost

**Pull request merged**